### PR TITLE
TDR-3680 - Clean up old validation code

### PR DESCRIPTION
**Pull request by vimleshtna**
Clean up old validation code as we will be using only metadata validation library now

### CODE REVIEW

File: test/controllers/util/FormFieldSpec.scala [hunk 1]
```diff
@@ -75,52 +75,6 @@ class FormFieldSpec extends AnyWordSpec with MockitoSugar with BeforeAndAfterEac
         .fieldErrors shouldBe List("alternativeDesc must be 8000 characters or less")
       RadioButtonGroupField.updateError(radioButtonGroupField, CLOSURE_STATUS_IS_MISSING).fieldErrors shouldBe List("CLOSURE_STATUS_IS_MISSING")
     }
-
```
Comment (thanhz) [inline test/controllers/util/FormFieldSpec.scala:78]: Unused dependency` org.apache.commons.lang3.NotImplementedException` and method ` def getDate(dateTime: LocalDateTime)`

File: test/controllers/util/DynamicFormUtilsSpec.scala [hunk 2]
```diff
@@ -77,318 +77,6 @@ class DynamicFormUtilsSpec extends AnyFlatSpec with MockitoSugar with BeforeAndA
     thrownException.getMessage should equal(s"$fieldTypeWithIncorrectPrefix is not a supported field type.")
   }
```
Comment (thanhz) [inline test/controllers/util/DynamicFormUtilsSpec.scala:79]: Unused dependency `import java.time.LocalDateTime`

**Review by thanhz — COMMENTED** [review]
Thanks, looks good. Just some minor comments and a question regarding the caching.

If we add/remove fields from the form will the users see these changes since the cache never expires?

**Comment by vimleshtna** [comment]
> Thanks, looks good. Just some minor comments and a question regarding the caching.
> 
> If we add/remove fields from the form will the users see these changes since the cache never expires?

Yeah, it should work because I have done the caching over the custom metadata, not the user data so it should work.

**Review by thanhz — APPROVED** [review]
Looks good, thanks

**Pull request merged**